### ISSUE TITLE
Failed to call method

### ISSUE THREAD

**Issue by akaFalsh** [stevengj/PyCall.jl#137]
I am trying to call method from class that was returned by factory class. However I get an error:
```
ERROR: `getindex` has no method matching getindex(::Function, ::Symbol)
```

It seems that it can not find a function that I am trying to call.

Example:
```
@pyimport model
ModelFactoryClass = model.pymember("ModelFactory")
modelfile = "/path/to/model.mod"
model = ModelFactoryClass[:open](modelfile)
dd = org_model[:getDataDescription]
numcells = dd[:getNumberOfCells]
```
org_model[:getDataDescription] - returns DataDescription class which has function getNumberOfCells.

**Comment by stevengj** [comment]
The problem is is probably that one of your functions is returning a class, and a class gets converted into a constructor function by default in PyCall, whereas apparently you want to call a static member?

Anyway, the workaround should be easy: just change `:` to `"`, e.g. `org_model["getDataDescription"]`, which tells PyCall that you want to return the raw PyObject and not a Function wrapper.

(This will be fixed by #101 once Julia 0.4 is available.)

**Comment by akaFalsh** [comment]
getDataDescription call went ok. However when I try to call getNumberOfCells in the same manner it returns another error:
```
numcells = dd[:getNumberOfCells]
ERROR: key not found: "getNumberOfCells"

numcells = dd["getNumberOfCells"]
ERROR: key not found: "getNumberOfCells"

numcells = dd.getNumberOfCells()
ERROR: type PyObject has no field getNumberOfCells
```

The description of an object dd says:
```
PyObject <bound method ModelPtr.getDataDescription of <model.ModelPtr; proxy of <Swig Object of type 'org::model::ModelPtr *' at 0x8d5a180> >>
```

Is there any method to see the detailed description of an object? With included methods or smb similar.
